app/web/jest.config.ts

```diff
@@ -0,0 +1,61 @@
+import nextJest from "next/jest";
+import type { Config } from "jest";
+
+// Providing the path to your Next.js app which will enable loading next.config.js and .env files
+const createJestConfig = nextJest({ dir: "./" });
+
+const customJestConfig: Config = {
+  verbose: true, // Shows detailed test results
```

Comment:
It was driving me nuts I couldn't see which tests failed when a lot ran at once.